Updating the decision guide for data processing in Fabric to include eventstreams (#1568)
MicrosoftDocs/fabric-docs: docs/fundamentals/decision-guide-pipeline-dataflow-eventstream-spark.md

## Copy activity, dataflow, Eventstream, and Spark properties

| | **Pipeline copy activity** | **Dataflow Gen 2** | **Eventstream** | **Spark** |
|---|:---:|:---:|:---:|:---:|
| **Use case** | Data lake and data warehouse migration,<br>data ingestion,<br>lightweight transformation | Data ingestion,<br>data transformation,<br>data wrangling,<br>data profiling | event data ingestion,<br>event data transformation | Data ingestion,<br>data transformation,<br>data processing,<br>data profiling |
| **Primary developer persona** | Data engineer,<br>data integrator | Data engineer,<br>data integrator,<br>business analyst | Data engineer,<br>data scientist,<br>data developer | Data integrator,<br>data engineer |
| **Primary developer skill set** | ETL,<br>SQL,<br>JSON | ETL,<br>M,<br>SQL | SQL, JSON, messaging |Spark (Scala, Python, Spark SQL, R) |
| **Code written** | No code,<br>low code | No code,<br>low code | No Code, <br>low code | Code |
| **Data volume** | Low to high | Low to high | Medium to High | Low to high |
| **Development interface** | Wizard,<br>canvas | Power query | Canvas | Notebook,<br>Spark job definition |

> | **Sources** | 30+ connectors | 150+ connectors | Database CDC, Kafka, Pub/Sub Messaging Systems, Event streams | Hundreds of Spark libraries |

Reviewer: ```suggestion
| **Sources** | 30+ connectors | 150+ connectors | Database supporting CDC (Change Data Capture), Kafka, Messaging Systems that support publish and subscribe pattern, Event streams | Hundreds of Spark libraries |
```

> | **Sources** | 30+ connectors | 150+ connectors | Database CDC, Kafka, Pub/Sub Messaging Systems, Event streams | Hundreds of Spark libraries |

Reviewer 2: It would be nice to have a link to the following article that lists all the supported sources. 
https://learn.microsoft.com/en-us/fabric/real-time-intelligence/event-streams/add-manage-eventstream-sources?pivots=enhanced-capabilities#supported-sources

Please remove "Event streams" from the list of sources.

> | **Destinations** | 18+ connectors | Lakehouse,<br>Azure SQL database,<br>Azure Data explorer,<br>Azure Synapse analytics | Eventhouse, Lakehouse, Activator Alert, Custom Endpoint | Hundreds of Spark libraries |

Reviewer 2: Here is the article that has the list of supported destinations:
https://learn.microsoft.com/en-us/fabric/real-time-intelligence/event-streams/create-default-derived-streams

Please add Derived stream to the list in this article. Thanks.

Author: to the list where?

## Scenario3

> Prashant is a data integrator with a deep knowledge of the business processes and systems. An upstream team has successfully exposed event data from business applications as messages that can be consumed through downstream systems. Prashant has been tasked with bringing this data into Microsoft Fabric for business decision support in real time. Prashant has been tasked with integrating this data into business processes that take action and transform the data as fast as possible.

Reviewer: ```suggestion
Prashant, a data integrator with deep expertise in business processes and systems. An upstream team has successfully exposed event data from business applications as messages that can be consumed through downstream systems. Prashant has been assigned to integrate event data from business applications into Microsoft Fabric for real-time decision support.
```

> The data volume is in the medium to high range and the organization does not want to leverage pro-code solutions. Prashant does not want to worry about needing to schedule the data to be extracted, and wants to simply forward events as they occur. Prashant decides that **Eventstreams** in Microsoft Fabric makes sense to achieve this goal. The eventstreams feature in the Microsoft Fabric Real-Time Intelligence experience lets you bring real-time events into Fabric, transform them, and then route them to various destinations without writing any code.

Reviewer: ```suggestion
Given the medium to high data volume and the organization's preference for no-code solutions, Prashant seeks a way to seamlessly forward events as they occur without managing extraction schedules. To meet this need, he chooses **Eventstreams** in Microsoft Fabric. Eventstreams within the Real-Time Intelligence experience enables real-time data ingestion, transformation, and routing to various destinations—all without writing any code.
```
